pkg/network/ndmz/dualstack.go

```diff
@@ -267,3 +284,21 @@ func waitIP6() error {
 	}
 	return nil
 }
+
+func stopBackgroundProbe() error {
+	probe, err := dhcp.NewBackgroundProbe(DMZPub4)
+	if err != nil {
+		return err
+	}
+
+	running, err := probe.IsRunning()
+	if err != nil {
+		return nil
+	}
+
+	// this means this process is already running, stop here
+	if running {
```

Comment:
This method is to 'stop' background probe, so the check should be `if !running` not `if running` !